server/compile_lexicon_to_json.py

```diff
@@ -126,19 +144,33 @@ def process_row(row, json_words, json_syllables, rows_of_crossid):
 def sort_row_tuples(row_tuples, reconstructed_sense):
     # first step: sorting by languages
     # 1) sort by languages alphabetically
-    row_tuples_sorted_by_languages = sorted(row_tuples, key = lambda x: x[1]['DOCULECT'])
+    row_tuples_sorted_by_languages = sorted(row_tuples, key=lambda x: x[1]["DOCULECT"])
     # 2) raise the languages we want to raise
-    ob = [(syl, row) for (syl, row) in row_tuples_sorted_by_languages if row['DOCULECT'] == 'Old_Burmese']
-    rangoon = [(syl, row) for (syl, row) in row_tuples_sorted_by_languages if row['DOCULECT'] == 'Rangoon']
-    remaining = [(syl, row) for (syl, row) in row_tuples_sorted_by_languages if (row['DOCULECT'] != 'Old_Burmese' and row['DOCULECT'] != 'Rangoon')]
+    ob = [
+        (syl, row)
+        for (syl, row) in row_tuples_sorted_by_languages
+        if row["DOCULECT"] == "Old_Burmese"
+    ]
+    rangoon = [
+        (syl, row)
+        for (syl, row) in row_tuples_sorted_by_languages
+        if row["DOCULECT"] == "Rangoon"
+    ]
+    remaining = [
+        (syl, row)
+        for (syl, row) in row_tuples_sorted_by_languages
+        if (row["DOCULECT"] != "Old_Burmese" and row["DOCULECT"] != "Rangoon")
+    ]
     row_tuples_sorted_by_languages = ob + rangoon + remaining
 
     # second step: sorting by senses
-    row_tuples_sorted_by_senses = sorted(row_tuples_sorted_by_languages, key = lambda x: x[1]['CONCEPT'])
+    row_tuples_sorted_by_senses = sorted(
+        row_tuples_sorted_by_languages, key=lambda x: x[1]["CONCEPT"]
+    )
     return row_tuples_sorted_by_senses
 
 
-def compile_to_json_full_cognates(path, proto, cognates="cogid"):
+def compile_to_json_full_cognates(path, proto, cognates="COGID"):
```

Comment:
Having talked with Nathan, I learned that we can exclude Proto-Germanic: this would mean we should exclude the proto-language param `proto` here as well.